content/10.diagnostics.md

```diff
@@ -1,33 +1,53 @@
 ## Diagnostics
 
+Identifying individuals who have contracted the infection is crucial to slow down the global pandemic.
 Given the high transmissibility of SARS-CoV-2, the development of reliable assays to detect SARS-CoV-2 infection even in asymptomatic carriers is vitally important.
 For instance, the deployment of wide-scale diagnostic testing followed by the isolation of infected people has been a key factor in South Korea's successful strategy for controlling the spread of the virus.
 Following the first release of the genetic sequence of the virus by Chinese officials on January 10 2020, the first test was released about 13 days later [@doi:10.2807/1560-7917.ES.2020.25.3.2000045].
-Identifying individuals who have contracted the infection is crucial to slowing down the global pandemic.
 The assays available to date use the following approaches to identify the active virus in patient samples.
 A range of diagnostic approaches from a methodological standpoint are being or could possibly be developed.
 
 ### Molecular Tests
 
-Molecular tests are often carried out from nasopharyngeal cavity samples, including throat wash and saliva [@doi:10/ggp4qx].
+Molecular tests are used to identify distinct genomic subsequences of a viral molecule in a sample.
+This first requires getting a patient sample from somewhere the virus is expected to be present in an infected individual.
+Common sources for a sample used in a molecular test include nasopharyngeal cavity samples, including throat wash and saliva [@doi:10/ggp4qx], or stool samples [@doi:10.1002/jmv.25742].
+Given a sample from a patient, molecular tests involve a number of steps to analyze a sample and produce results.
+These steps include sample pre-processing, followed by library preparation, and then sequencing itself [doi:10.1016/j.biotechadv.2020.107537].
+When testing for RNA viruses like SARS-CoV-2, pre-processing is needed in order to create DNA, which is easier to process and sequence, from the initial RNA sample.
```

Comment:
```suggestion
When testing for RNA viruses like SARS-CoV-2, pre-processing is needed in order to create DNA, which can then be replicated during PCR, from the initial RNA sample.
```
It's more than that it's easier to process, it's just that the way PCR works is to get DNA replication going!